app/controllers/shift_commerce/bundles_controller.rb

```diff
@@ -23,7 +23,8 @@ class BundlesController < ::ApplicationController
         'canonical_path',
         'asset_files',
         'bundle_groups',
-        'meta_attributes'
+        'meta_attributes',
+        'updated_at'
```

Comment:
Have noticed that bundle caching was not working, because of missing updated_at value